packages/shared/src/hooks/usePostById.ts

```diff
@@ -169,7 +172,7 @@ const usePostById = ({ id, options = {} }: UsePostByIdProps): UsePostById => {
       post: post?.post,
       relatedCollectionPosts: post?.relatedCollectionPosts,
       isError,
-      isLoading,
+      isLoading: !post?.post ?? isLoading,
```

Comment:
This is a bit weird, but we also not loading is we have initialData basically (from the actual post return)

Reply:
`!post?.post` will always be boolean so `??` will never evaluate, I think it needs:
```suggestion
      isLoading: !post?.post || isLoading,
```